operations/services.md

```diff
@@ -108,31 +111,50 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | :-- | :-- | :-- | :-- |
 | `Id` | string | required | Unique identifier of the service. |
 | `EnterpriseId` | string | required | Unique identifier of the [Enterprise](enterprises.md#enterprise). |
-| `CreatedUtc` | string | required | Creation date and time of the service in UTC timezone in ISO 8601 format. |
-| `UpdatedUtc` | string | required | Last update date and time of the service in UTC timezone in ISO 8601 format. |
 | `IsActive` | boolean | required | Whether the service is still active. |
 | `Name` | string | required | Name of the service. |
 | `Options` | [Service options](#service-options) | required | Options of the service. |
+| `Ordering` | integer | required | Order value for presentation purposes. |
 | `Data` | [Service data](#service-data) | required | Additional information about the specific service. |
-| `ExternalIdentifier` | string | optional, max 255 characters | Identifier of the service from external system. |
+| `ExternalIdentifier` | string | optional, max length 255 characters | Identifier of the service from external system. |
+| `CreatedUtc` | string | required | Creation date and time of the service in UTC timezone in ISO 8601 format. |
+| `UpdatedUtc` | string | required | Last update date and time of the service in UTC timezone in ISO 8601 format. |
+| ~~`StartTime`~~ | string | optional | This property is deprecated. |
```

Comment:
```suggestion
| ~~`StartTime`~~ | ~~string~~ | ~~optional~~ | **Deprecated!** |
```
I think it should be like that, when something is deprecated 🤔 Apply to all deprecations here

For example
https://github.com/MewsSystems/gitbook-connector-api/blob/ca239e35e0216c341e77cb39d642d88efeb8e704/operations/accountingitems.md?plain=1#L246

cc @jnv

Reply:
Changed it.

Reply:
@VBuben is correct, we should use this standard form for how we label something as Deprecated throughout the docs. However, in addition it should still have a description! See the example above.